package.json

```diff
@@ -35,6 +35,7 @@
     "d3": "^3.5.6",
     "focus-trap-react": "^3.1.4",
     "glamor": "^2.20.40",
+    "js-logger": "^1.6.1",
```

Comment:
Why was this added? Doesn't seem related.

Reply:
So whenever I try to do an `npm link` without it installed, the build fails on the serenity side. This was just a simple fix and felt like it didn't harm to have it in the dev-deps. I know that other people have run into the same issue.

I can dig deeper however and see if there is something going on that I don't realize.

Reply:
@TayteStokes has run into this issue before. Our solution in the past has been to install it, then remove it. However @DrewWarrenTIY says that he hasn't run into the issue with `js-logger` while linking.

Essentially what I do
- run `npm link` in `mx-react-components`
- run `npm dev-release` in `mx-react-components`
- run `npm link mx-react-components` in `serenity`
- run `npm run dev` in `serenity.

Then I get these errors: 
```
ERROR in ../mx-react-components/dist/constants/IconPaths.js
Module not found: Error: Can't resolve 'js-logger' in '/Users/jacob.pettingill/repos/mx-react-components/dist/constants'
 @ ../mx-react-components/dist/constants/IconPaths.js 3:13-33
 @ ../mx-react-components/dist/index.js
 @ ./src/components/app/IEDeprecationModal.js
 @ ./src/components/app/AppWrapper.js
 @ ./src/widgetLoader.js
 @ multi ./src/widgetLoader.js
```

I can't seem to nail down why this error is occurring. @sambev @phantomxc any ideas on why this might be happening?

The simple solution would be to either install `js-logger` as a `dev-dependency` or else install it while I test and then remove it. But I would like to nail down a longer term solution.

Reply:
I have tried `npm ci`, and removing the modules. As well as a complete refresh on the linking several times.

Reply:
It looks like it might have something to do with webpack configs. I guess symlink resolution is a difficult problem to resolve. https://github.com/webpack/webpack/issues/985. I am going to just install temporarily for now.

Reply:
I think it would be best to investigate removing js-logger entirely. Is it being used?